Competition2017/src/competition/subsystems/shoot_fuel/FireLeftTracerRoundsCommandGroup.java

```diff
@@ -0,0 +1,29 @@
+package competition.subsystems.shoot_fuel;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.agitator.AgitatorsManagerSubsystem;
+import competition.subsystems.agitator.commands.IntakeAgitatorCommand;
+import competition.subsystems.shooter_belt.ShooterBeltsManagerSubsystem;
+import competition.subsystems.shooter_belt.commands.RunBeltSlowlyUsingPowerCommand;
+import competition.subsystems.shooter_wheel.ShooterWheelsManagerSubsystem;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class FireLeftTracerRoundsCommandGroup extends CommandGroup {
```

Comment:
Make a single command "FireTracerRoundsCommandGroup" which has a `setSide` method which allows you to select the side that it runs on. That means that we don't have to make changes to both instances of this command (or, more likely, forget to!).